README.md

```diff
@@ -25,6 +25,11 @@ In a PowerShell window run the following command to create an AWS profile for us
 Login-AWSSAML -ProfileName 'Name'
 ```
 
+You can specify a different session duration.
+```powershell
+Login-AWSSAML -SessionDuration 43200
```

Comment:
Based on the documentation you can't have a value over 1 hour...  I'm curios if you have tested this and if you got back a valid token that worked for more than 1 hour?

AWS Docs: https://docs.aws.amazon.com/powershell/latest/reference/items/Use-STSRoleWithSAML.html

Reply:
Thank you @eelkram for the feedback! I am referrring to the following document.

https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithSAML.html

> You can provide a DurationSeconds value from 900 seconds (15 minutes) up to the maximum session duration setting for the role. This setting can have a value from 1 hour to 12 hours.

And the duration of the session of the role I assume is configured to be 12 hours. I tested and it worked for me.

Reply:
Awesome, that is great to hear.